repo-scripts/changelog-generator/tsconfig.json

```diff
@@ -3,7 +3,8 @@
     "strict": true,
     "outDir": "dist",
     "lib": [
-      "ESNext"
+      "ESNext",
+      "dom"
```

Comment:
Why is this file changed?

Reply:
Without this, adding the @types/dom-chromium-ai dependency causes the postsubmit script to fail. I'll try reproing the issue on a stand-alone branch.

Reply:
Created PR https://github.com/firebase/firebase-js-sdk/pull/8916 to repro in CI env.